roles/ovirt-repositories/README.md

```diff
@@ -0,0 +1,69 @@
+
+oVirt Repositories
+==================
+
+The `oVirt.repositories` role is used to set the repositories required for oVirt engine installation.
+By default it copies content of /etc/yum.repos.d/ to /tmp/repo-backup-{{timestamp}}.
```

Comment:
By default it copies content of /etc/yum.repos.d/ to /tmp/repo-backup-{{timestamp}} so it's easy to undo that operation.

Reply:
Fixed.